Mage.Sets/src/mage/cards/t/ThijarianWitness.java

```diff
@@ -0,0 +1,85 @@
+package mage.cards.t;
+
+import mage.MageInt;
+import mage.abilities.common.DiesCreatureTriggeredAbility;
+import mage.abilities.effects.common.ExileTargetEffect;
+import mage.abilities.effects.keyword.InvestigateEffect;
+import mage.abilities.keyword.FlashAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.SubType;
+import mage.filter.FilterPermanent;
+import mage.filter.common.FilterCreaturePermanent;
+import mage.filter.predicate.Predicate;
+import mage.filter.predicate.mageobject.AnotherPredicate;
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+import mage.watchers.common.AttackingBlockingWatcher;
+
+import java.util.UUID;
+
+/**
+ * @author notgreat
+ */
+public final class ThijarianWitness extends CardImpl {
+
+    private static final FilterPermanent filter
+            = new FilterCreaturePermanent("another creature");
+
+    static {
+        filter.add(AnotherPredicate.instance);
+        filter.add(AttackBlockAlonePredicate.instance);
+    }
+
+    public ThijarianWitness(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.CREATURE}, "{1}{G}");
+
+        this.subtype.add(SubType.ALIEN);
+        this.subtype.add(SubType.CLERIC);
+        this.power = new MageInt(0);
+        this.toughness = new MageInt(4);
+
+        // Flash
+        this.addAbility(FlashAbility.getInstance());
+
+        DiesCreatureTriggeredAbility ability = new DiesCreatureTriggeredAbility(
```

Comment:
no rules text comment? are you not using gen-card.pl?

Reply:
Truthfully no. Since I'm usually only implementing a few complicated cards I haven't bothered with it in the past. I probably should though.